src/v2/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/UploadPhotos.tsx

```diff
@@ -25,6 +27,7 @@ export const UploadPhotos: React.FC = () => {
     },
   } = useRouter()
 
+  const [isSubmissionApiError, setIsSubmissionApiError] = useState(false)
```

Comment:
here the same)

Reply:
for primitive types, we don't need to type explicitly when using useState. typescript already infers it from initial state.